src/Tribe/Venue.php

```diff
@@ -698,7 +698,9 @@ public static function get_fetch_callback( $event ) {
 		}
 
 		return static function () use ( $event ) {
-			$venue_ids = array_map( 'absint', (array) get_post_meta( $event, '_EventVenueID' ) );
+			$venue_ids = array_filter(
+				array_map( 'absint', (array) get_post_meta( $event, '_EventVenueID' ) )
```

Comment:
* :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed
* :no_entry_sign: A cast statement must not be followed by a space
* :warning: Line exceeds 80 characters; contains 87 characters

<sub>hash: 4cd6f653a69cc6558eafe538499dd7de</sub>